docs/react-testing-library/example-intro.mdx

```diff
@@ -4,6 +4,65 @@ title: Example
 sidebar_label: Example
 ---
 
+## Quickstart
+
+This is a very simple setup to get you started. 
+If you want to see a description of what each line does, 
+scroll down to the [annotated version](#quickstart-annotated).
+Scroll down to [Full Example](#full-example) to see a more advanced test setup.
+
+```jsx
+import {render, fireEvent, waitFor, screen} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Fetch from '../fetch'
+
+test('loads and displays greeting', async () => {
+
+  // ARRANGE
+  render(<Fetch url="/greeting" />)
+  
+  // ACT
+  fireEvent.click(screen.getByText('Load Greeting'))
+  await waitFor(() => screen.getByRole('heading'))
+  
+  // ASSERT
+  expect(screen.getByRole('heading')).toHaveTextContent('hello there')
+  expect(screen.getByRole('button')).toBeDisabled()
+
+})
+```
+
+<details id="quickstart-annotated">
+  <summary>Quickstart (Annotated Example)</summary>
+
+  ```jsx
+  // import react-testing methods
+  import {render, fireEvent, waitFor, screen} from '@testing-library/react'
+  // add custom jest matchers from jest-dom
+  import '@testing-library/jest-dom'
+  // the component to test
+  import Fetch from '../fetch'
+
+  test('loads and displays greeting', async () => {
+
+    // Render a React element into the DOM
+    render(<Fetch url="/greeting" />)
+    
+    fireEvent.click(screen.getByText('Load Greeting'))
```

Comment:
```suggestion
  // import react-testing methods
  import {render, waitFor, screen} from '@testing-library/react'
  // user-event provides methods for testing interactions
  import userEvent from '@testing-library/user-event'
  // add custom jest matchers from jest-dom
  import '@testing-library/jest-dom'
  // the component to test
  import Fetch from '../fetch'

  function setup(jsx) {
    return {
      // create a user-event instance
      user: userEvent.setup(),
      // render a React element
      ...render(jsx),
    }
  }

  test('loads and displays greeting', async () => {

    // Render the component
    const { user } = setup(<Fetch url="/greeting" />)
    
    await user.click(screen.getByText('Load Greeting'))
```

Reply:
@ph-fritsche I decided to drop the `setup` version since it's available in version 14. Right now, most of us are using Create-react-app to start working on new React projects and the current CRA setup is still using v13 so the setup example will be breaking when beginners try to run the test with setup on their installations.